vnext/ReactUWP/Modules/AppThemeModuleUwp.cpp

```diff
@@ -43,19 +55,52 @@ AppTheme::AppTheme(const std::shared_ptr<IReactInstance>& reactInstance, const s
   });
 }
 
+// Returns the RBG values for the 8 relevant High Contrast elements.
+folly::dynamic AppTheme::getHighContrastColors() {
+  winrt::Windows::UI::Color ButtonFaceColor = m_uiSettings.UIElementColor(winrt::Windows::UI::ViewManagement::UIElementType::ButtonFace);
```

Comment:
>winrt::Windows::UI::ViewManagement [](start = 74, length = 34)

nit: Probably personal preference, but it's a little too long for me. You could add Windows::UI::ViewManagement to namespace winrt at the too and just do "winrt::UIElementType::ButtonFace"